docs/src/main/asciidoc/hibernate-orm-panache-guide.adoc

```diff
@@ -292,42 +292,42 @@ NOTE: You can also write your queries in plain
 link:https://docs.jboss.org/hibernate/orm/5.4/userguide/html_single/Hibernate_User_Guide.html#hql[HQL]:
 
 [source,java]
---
+----
 Order.find("select distinct o from Order o left join fetch o.lineItems");
---
+----
 
 
 == Query parameters
 
-You can pass query parameters by index (1-based):
+You can pass query parameters by index (1-based) as shown below:
 
 [source,java]
---
+----
 Person.find("name = ?1 and status = ?2", "stef", Status.Alive);
---
+----
 
 Or by name using a `Map`:
 
 [source,java]
---
+----
 Map<String, Object> params = new HashMap<>();
 params.put("name", "stef");
 params.put("status", Status.Alive);
 Person.find("name = :name and status = :status", params);
---
+----
 
-Or using the convenience class `Parameters` to either build a `Map` or just use as-is:
+Or using the convenience class `Parameters` to either build a `Map` or just used as-is:
```

Comment:
Maybe the following would be better? (we would have to change it in the MongoDB doc too)
```suggestion
Or using the convenience class `Parameters` either as is or to build a `Map`:
```